assets/scss/_explorer.scss

```diff
@@ -40,28 +40,42 @@
   main.content {
     @include xy-cell($size: 9.5, $gutter-position: 'left');
 
+    .pixelsets-wrapper {
+      @include table-scroll;
+    }
+
     table.pixelsets {
       @include table;
       @include table-hover;
 
+      width: 200%;
+
+      .pixelset {
+        cursor: pointer;
+      }
+
       td {
         vertical-align: top;
+      }
 
-        &.filename {
-          font-family: $font-family-monospace;
-        }
+      .filename {
+        font-family: $font-family-monospace;
+      }
 
-        &.analysis,
-        &.experiments {
-          font-size: 0.8rem;
-        }
+      .analysis,
+      .experiments {
+        font-size: 0.8rem;
+      }
```

Comment:
Limiting nested levels optimizes page load time?

Reply:
nope, it improves readability 😛